### PR TITLE
Bugfix: iterating over the items() view of a section breaks

### PR DESCRIPTION
**Pull request by ctheune**

### CODE REVIEW

**Comment by FlorianWilhelm** [comment]
Hi, this is not a bugfix but rather a new feature, right?

File: src/configupdater/configupdater.py [hunk 1]
```diff
@@ -393,6 +393,15 @@ def set(self, option, value=None):
             self.__setitem__(option, value)
         return self
 
+    def items(self):
+        """Return a list of (name, value) tuples for each option in
+        this section.
+
+        Returns:
+            list: list of :class:`Section` or :class:`Option` objects
```
Comment (FlorianWilhelm) [inline src/configupdater/configupdater.py:401]: This is the wrong return type. It's a list of tuples.

**Comment by ctheune** [comment]
> Hi, this is not a bugfix but rather a new feature, right?

It's a bugfix. You're inheriting MutableMapping and that already provides an iter()-View for items() which is currently broken. Also, sections do provide a working way of iterating over items(), so ... I'd call it a bugfix. :)

File: src/configupdater/configupdater.py (hunk 1 above)
Comment (ctheune) [reply src/configupdater/configupdater.py:401]: Meh. Copy/paste ... I'll fix that.

**Comment by FlorianWilhelm** [comment]
Good point and thanks for this PR!

**Comment by FlorianWilhelm** [comment]
Are there any other things you have in progress or can I just release a new bugfix version with this fix alone?

**Comment by ctheune** [comment]
No, I'm good, thanks! I just started using it but I already finished the integration with my project and am happy. :)

**Comment by ctheune** [comment]
FYI: Here's what I'm doing with it: https://github.com/flyingcircusio/batou/commit/b252f7a0b4f8d12cf5ea2e70c3fc3dad0ebaa516

**Pull request merged**